manifests/cf-rabbitmq-template.yml

```diff
@@ -8,12 +8,22 @@ releases:
   version: 0.179.0
```

Comment:
Given that this is required for v0.180+, I think it would make sense to update this version too.